accepted/003-seq-feature.md

```diff
@@ -0,0 +1,199 @@
+---
+Feature name: interval-metadata
+Start date: <2015-12-18>
+Pull request: 1
+Authors: ["@RNAer", "@mortonjt"]
+Contributors: ["@RNAer", "@mortonjt", "@ebolyen", "@jairideout", "@wasade", "@gregcaporaso", "@rob-knight"]
```

Comment:
remove the authors from the contributors list (each name should only go in one of the lists).